README.md

```diff
@@ -1,54 +1,147 @@
-# ACME webhook example
+[![codecov](https://codecov.io/gh/toothstone/cert-manager-webhook-anexia/branch/main/graph/badge.svg?token=G573SD12QG)](https://codecov.io/gh/toothstone/cert-manager-webhook-anexia)
 
-The ACME issuer type supports an optional 'webhook' solver, which can be used
-to implement custom DNS01 challenge solving logic.
+# cert-manager ACME webhook for Anexia CloudDNS
 
-This is useful if you need to use cert-manager with a DNS provider that is not
-officially supported in cert-manager core.
+This repository provides a webhook to use cert-manager with Anexia CloudDNS.
 
-## Why not in core?
+## Requirements
+-   [go](https://golang.org/) >= 1.13.0
+-   [helm](https://helm.sh/) >= v3.0.0
+-   [cert-manager](https://cert-manager.io/) >= 1.6.0
 
-As the project & adoption has grown, there has been an influx of DNS provider
-pull requests to our core codebase. As this number has grown, the test matrix
-has become un-maintainable and so, it's not possible for us to certify that
-providers work to a sufficient level.
+## Installation
 
-By creating this 'interface' between cert-manager and DNS providers, we allow
-users to quickly iterate and test out new integrations, and then packaging
-those up themselves as 'extensions' to cert-manager.
+### cert-manager
 
-We can also then provide a standardised 'testing framework', or set of
-conformance tests, which allow us to validate the a DNS provider works as
-expected.
+Follow the [instructions](https://cert-manager.io/docs/installation/) in the cert-manager documentation to install it within your cluster.
 
-## Creating your own webhook
+### Webhook
 
-Webhook's themselves are deployed as Kubernetes API services, in order to allow
-administrators to restrict access to webhooks with Kubernetes RBAC.
+#### Using public helm chart
+```bash
+helm repo add cert-manager-webhook-anexia https://toothstone.github.io/cert-manager-webhook-anexia
+helm install --namespace cert-manager cert-manager-webhook-anexia cert-manager-webhook-anexia/cert-manager-webhook-anexia
+```
 
-This is important, as otherwise it'd be possible for anyone with access to your
-webhook to complete ACME challenge validations and obtain certificates.
+#### From local checkout
 
-To make the set up of these webhook's easier, we provide a template repository
-that can be used to get started quickly.
+```bash
+helm install --namespace cert-manager cert-manager-webhook-anexia deploy/cert-manager-webhook-anexia
+```
+**Note**: The kubernetes resources used to install the webhook should be deployed within the same namespace as the cert-manager.
```

Comment:
Should as in RFC2119? If yes: see RFC8174 and make it uppercase, if not: please document what happens if not deployed in same namespace as cert-manager, what has to be done to have it work, .. at least a rough overview (or a "should (not uppercase) work, but never tested - doc updates welcome")

Reply:
AFAIK it is a "should" as in "this is what we support, putting them somewhere else might work with some intervention, ymmv" - I've not actually tried installing it somewhere else.

We can change it to something like "Only installation to the same namespace as the cert-manager is supported and tested." if you would like to avoid above vagueness?

Reply:
I would strip the "supported", saying only same namespace as cert-manager is tested and thus we recommend that

but yeah, basically this :)